src/main/java/de/dennisguse/opentracks/MarkerListActivity.java

```diff
@@ -237,9 +237,6 @@ private boolean handleContextItem(int itemId, long... longMarkerIds) {
                 }
                 return true;
             case R.id.list_context_menu_delete:
-                if (markerIds.length > 1 && markerIds.length == viewBinding.markerList.getCount()) {
```

Comment:
markerIds must be null, so that DeleteMarkerDialogFragment will show the "delete all message".
Don't know if the current is code is working without this if - markerIds might be an initialized array with length=0.
And then the deletion will not happen anymore.

Reply:
If markerIds is null then OT crashes. Look at 82 DeleteMarkerDialogFragment's line (it uses markerIds but it can be null):

```java
for (Marker.Id markerId : markerIds) {
    contentProviderUtils.deleteMarker(getContext(), markerId);
}
```

All that code was only for show "Delete all markers?" instead of "Delete selected?" in the title of the Dialog.

Reply:
In my last commit is my proposal (it can be undo :D)